diff -Naur lxqt-globalkeys-0.14.3/daemon/core.cpp lxqt-globalkeys-0.14.3.tpg/daemon/core.cpp
--- lxqt-globalkeys-0.14.3/daemon/core.cpp	2019-10-16 15:17:40.000000000 +0000
+++ lxqt-globalkeys-0.14.3.tpg/daemon/core.cpp	2020-03-10 09:27:43.182713614 +0000
@@ -33,6 +33,7 @@
 #include <QTimer>
 #include <QDBusConnectionInterface>
 #include <QDBusServiceWatcher>
+#include <QFile>
 
 #include <cstddef>
 #include <cstdlib>
